supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -9,17 +9,7 @@ Bold all graphical user interface elements, including, but not limited to, menu
 
 .Example AsciiDoc
 ----
-Click *OK*.
-----
-
-[[menu-navigation]]
-== Menu navigation
-
-Bold menu names and join menu items with an arrow to indicate a sequence of actions.
-
-.Example AsciiDoc
-----
-Click *Design* -> *Projects*.
+Click *Metering* on the *Installed Operators* page.
```

Comment:
@bergerhoffer I see two issues now on this page. 

- The button example also should be coded using the :experimental: btn macro, and so that should also be removed.
- The current menu example is no longer an example of a menu item.

I think you should just remove the examples from this page until we arrive at a consensus regarding the correct way to code them. Anything else will, to paraphrase Mayor Daley at the 1968 Democratic convention, just preserve confusion.

Reply:
@stoobie I'm not sure what you mean. This update removed the menu item completely, and got rid of button as an example GUI element, as you suggested, to avoid the topic of whether to use the :experimental: markup or not.

You can preview the file here: https://github.com/redhat-documentation/supplementary-style-guide/blob/d2fd117869466b95e95c428cb06de668d83b2707/supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

Reply:
@bergerhoffer Oops. Sorry, I see that you're right. I don't know why I didn't see the updates.

Reply:
@stoobie No worries! If this looks good to you, can you dismiss the changes requested and approve the PR?